.github/workflows/verify-release-label-added.yml

```diff
@@ -10,9 +10,10 @@ name: Verify Release Label Added
       - unlabeled
     branches:
       - dev
+  merge_group:
```

Comment:
> You can use the merge_group event to trigger your GitHub Actions workflow when a pull request is added to a merge queue [docs](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue#triggering-merge-group-checks-with-github-actions)

similar to our "Linting and Unit/e2e Tests" workflow [here](https://github.com/grafana/oncall/blob/dev/.github/workflows/linting-and-tests.yml#L12)